packages/client/src/utils/draftUtils.ts

```diff
@@ -203,3 +207,13 @@ export function isDeclarationInReadyToReviewStatus(
     submissionStatus
   )
 }
+
+export function getRegisteringOfficeId(
+  declaration?: IPrintableDeclaration
+): string | null {
+  const registeringHistory = (
+    declaration?.data?.history as unknown as Record<string, any>[]
```

Comment:
I think this would be the correct type for this

https://github.com/opencrvs/opencrvs-core/blob/d8db049744694f87f514199859f8558774d1bb02/packages/client/src/utils/gateway.ts#L672